.markdownlint.yaml

```diff
@@ -2,12 +2,6 @@
 # Disable inline html linter is needed for <details> <summary>
 MD033: false
 
-# MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md013.md
-# Conforms to .clang-format ColumnLimit
-# Update the comment in .clang-format if we no-longer tie these two column limits.
-MD013:
```

Comment:
Should we also set `MD013: false` here? I noticed you set it on `.pre-commit-config.yaml`.

Reply:
As near as I've been able to figure out `MD013: false` doesn't actually work, it still ends up with the default line length of 80 being checked. The property isn't boolean, like the ones we have set to `false`.